js-core/homeworks/homework-2/src/main.js

```diff
@@ -0,0 +1,76 @@
+/* 1. Составить предложение из нижестоящих переменных:
+"Сколько нужно программистов чтобы сделать проект ? 1, 25, команда"
+*/
+
+let numbers = [25, 1];
+let project = 'проект';
+let team = `команда`;
+let howMuch = 'Сколько';
+let sentence = 'нужно программистов чтобы сделать проект ?';
+
+let task1 = howMuch + ' ' + sentence + ' ' + numbers[1] +  ', ' + numbers[0] + ', ' + team;
```

Comment:
what about es6 string? I guess it would look much better

Reply:
let task_v2 = `${howMuch} ${sentence} ${numbers[1]}, ${numbers[0]}, ${team}`;
console.log(task_v2);